TRACONBoundaries.geojson

```diff
@@ -250,6 +252,36 @@
+    {"type":"Feature","properties":{"id":"LTCV","prefix":["LTCV"],"suffix":"TWR","name":"Sirnak Tower/Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[41.824086111111114,37.640411944444445],[41.914524722222218,37.677803888888889],[41.993829722222223,37.692764722222222],[42.075739444444444,37.69696444444444],[42.153241111111107,37.687401666666666],[42.219780833333338,37.670622222222221],[42.314656666666664,37.628359444444442],[42.34884,37.601953055555555],[42.387503055555555,37.571891111111114],[42.451351944444447,37.482504444444444],[42.473519166666669,37.392789722222219],[42.47597861111111,37.357776388888894],[42.468814444444448,37.306194722222216],[42.460393611111115,37.277132222222221],[42.444443888888884,37.240727222222226],[42.426279722222219,37.206527222222228],[42.402493611111112,37.177365555555554],[42.375736111111109,37.149577777777779],[42.357221944444447,37.132478055555559],[42.324770938795318,37.10698748568656],[42.216667,37.258333],[42.083333,37.170833],[41.824358013769647,37.11055429784038],[41.796411111111112,37.123596944444444],[41.734358055555553,37.154095833333329],[41.713613055555555,37.1772625],[41.676398888888883,37.225539444444443],[41.657312222222224,37.265116111111112],[41.643332777777779,37.317818333333335],[41.640860555555555,37.395701388888888],[41.65228888888889,37.452632777777779],[41.67773722222222,37.505409166666666],[41.726511111111115,37.571163333333338],[41.7818625,37.614453611111109],[41.824086111111114,37.640411944444445]]]]}},
+    {"type":"Feature","properties":{"id":"LTCW","prefix":["LTCW"],"suffix":"TWR","name":"Yuksekova Tower/Approach"},"geometry":{"type":"MultiPolygon","coordinates":[[[[44.192728333333328,37.963382222222222],[44.248179089062958,37.959749139581824],[44.216558,37.899067],[44.449897,37.765731],[44.566567,37.749064],[44.583236,37.449058],[44.799908,37.299053],[44.749906,37.199053],[44.757369,37.166172],[44.630556,37.158333],[44.270833,36.933333],[44.166667,37.108333],[44.233333,37.241667],[44.033333,37.3],[43.945674718021856,37.231821141443035],[43.921683055555555,37.235266944444447],[43.909591846092127,37.238718543054731],[43.884502777777776,37.245880555555559],[43.884502777777776,37.245880555555559],[43.820665277777778,37.303418611111105],[43.7626,37.3762575],[43.724986388888894,37.476623611111116],[43.71512805555556,37.564753611111108],[43.73477611111111,37.664393888888888],[43.787612222222222,37.766934722222224],[43.850025277777782,37.830976111111113],[43.939704722222217,37.895360833333335],[44.024825277777779,37.933131388888889],[44.141081111111113,37.959177777777782],[44.192728333333328,37.963382222222222]]]]}}
```

Comment:
The file is misformatted and cannot be opened. I suspect this line is the issue as it is missing a comma at the end. Please make sure the file can be opened and resubmit.